backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/RabbitMQHealthCheck.java

```diff
@@ -52,7 +52,7 @@ public Result check() {
                 return Result.healthy(COMPONENT_NAME);
             } else {
                 String message = "The created connection was not opened";
-                LOGGER.error(message);
+                LOGGER.error(String.format("Unhealthy RabbitMQ instances: %s", message));
```

Comment:
`LOGGER.errorUnhealthy RabbitMQ instances: {}", message);`

slf4j have string interpolation built in